pandas/core/groupby.py

```diff
@@ -660,6 +627,8 @@ def __iter__(self):
         """
         return self.grouper.get_iterator(self.obj, axis=self.axis)
 
+    @Substitution(name='groupby')
+    @Appender(_doc_template)
```

Comment:
This adds a `See also` section, but there already is a see also section in this docstring (which links to other relevant functions). So I would just leave the appender out here)